### PR TITLE
count code

### PR DESCRIPTION
**Pull request by ericnatejones**

### CODE REVIEW

File: daily-challenges/count-code/index.test.js [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+describe('Returns the number of times "code" appears in given string, ignoring the "d', () => {
+    test("should return number of times 'co_e' exists in the string", () => {
+        expect(countCode("aaacodebbb")).toBe(1)
+        expect(countCode("cozexxcope")).toBe(2)
+        expect(countCode("codexxcode")).toBe(2)
+    })
+})
+
+xdescribe("Extra credit challenges", () => {
+    test("remove the x and make your function pass these edge cases", () => {
```
Comment (bobziroll) [inline daily-challenges/count-code/index.test.js]: These tests don't actually match the extra credit from the writeup. Change one or the other so they match. (If you change the writeup, make sure to change the `README.md` file here too)
